resources/assets/uad_lists.json

```
@@ -34356,5 +34356,501 @@
+    "removal": "Expert"
+  },
+  "com.itel.TWSService": {
+    "list": "Oem",
+    "description": "This app is used for connection to TECNO Hipods.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  "com.mediatek.schpwronoff": {
+    "list": "Misc",
+    "description": "Set schedule power on & off.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  "com.reallytek.wg": {
+    "list": "Oem",
+    "description": "Calibration, Test Camera things.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  "com.sagereal.lcmtest": {
+    "list": "Oem",
+    "description": "Secret code: 88. LCD, LCM Test.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  "com.sgrl.fmradio": {
+    "list": "Oem",
+    "description": "FM Radio\nSafe to remove if unused.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  "com.sgrl.pjj.factorymode": {
+    "list": "Oem",
+    "description": "FactoryMode, Calibration, Testing hardware things.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  "com.sprd.cameraipcontrol": {
+    "list": "Oem",
+    "description": "Secret code: 83785. CameraIPControl, has some camera features for testing.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  "com.sprd.firewall": {
+    "list": "Oem",
+    "description": "Blacklist Calls.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  "com.sprd.flashcontrol": {
+    "list": "Oem",
+    "description": "Flashlight Control, not sure how useful it is.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  "com.sprd.omacp": {
+    "list": "Oem",
+    "description": "OTA Config and Settings.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Expert"
+  },
+  "com.sprd.overlay.sprdnote": {
+    "list": "Oem",
+    "description": "Overlay to 'com.sprd.sprdnote' png icons.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  "com.sprd.providers.photos": {
+    "list": "Oem",
+    "description": "Photos Types Storage\nBokeh, portrait, FDR, AI photo.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  "com.spreadtrum.ims": {
+    "list": "Oem",
+    "description": "Needed for WiFi calling, VoLTE, VoWifi.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  "com.spreadtrum.vowifi.conf": {
+    "list": "Oem",
+    "description": "VoWifi config Secret Code: 869434234",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
```

```suggestion
    "removal": "Advanced"
```
If it can be used to configure something, it may be useful.

---

its hidden but we can set advanced

---

Alright, change it please.